src/github.ts

```diff
@@ -547,6 +582,59 @@ export class GitHub {
     return tags;
   }
 
+  async findMergedPullRequests(number?: number): Promise<MergedGitHubPR[]> {
+    const targetBranch = await this.getDefaultBranch();
+    const response = await this.graphqlRequest({
+      query: `query findMergedPullRequests($owner: String!, $repo: String!, $num: Int!) {
+          repository(owner: $owner, name: $repo) {
+            ref(qualifiedName: $targetBranch) {
+              target {
+                ... on Commit {
+                  history(first: $num) {
```

Comment:
I probably need to do some testing. 

If you merge or rebase/merge and all the commits are associated to the same PR, this method could return duplicates and could fill the entire page(s) of responses. We would likely still need to paginate the graphql query as well.

Reply:
I wonder if rather than paginating, there's a sort we can apply on the lookup, so the most recent interaction with the well known branch name is at the top? we should be able to test this in the sandbox.